denys-kravchuk/README.md

```diff
@@ -0,0 +1,73 @@
+# Social network for volunteers
+
+# Description
+This social network was created to help our volunteers and unite them on the one platform.
+
+# Presentation
+![chrome_0Z5R5Betcu](https://github.com/Denys-Kravchuk982910/op-kp23-Kravchuk/assets/114987963/0ab54bd1-dc09-4d57-9382-fa366ee5343d)
+
+![chrome_YgeWL5Y9A8](https://github.com/Denys-Kravchuk982910/op-kp23-Kravchuk/assets/114987963/1cb11d40-19f3-4a67-ae20-080e5ff75ab8)
+
+![chrome_S6wQlsydWS](https://github.com/Denys-Kravchuk982910/op-kp23-Kravchuk/assets/114987963/71e1498a-9125-4cf3-9567-be2b8a36ce07)
+
+
+
+# Get started
+1. Clone the server part: git clone https://github.com/Denys209818/DiplomaWork.git
+2. Clone the client part: https://github.com/Denys209818/DiplomaWorkFront.git
+3. you should publish these projects using embedded vs functions or just use special commands dotnet publish, npm publish.
+4. You should start these projects. Dotnet DiplomaWork.dll; npm start
+
+
+# For developers
+1. To build the project you should use the utilities like dotnet build or npm build(for frontend).
+2. To delpoy this project you should upload this project on KVM server which will execute this project.
```

Comment:
- To deploy this project, .... Missing comma.
- This sentence is too long. Break it into two sentences:.. on KVM server. The KVM server executes this project (use present tense instead of future).